fn_phish_tank/fn_phish_tank/components/fn_phish_tank_submit_url.py

```diff
@@ -0,0 +1,92 @@
+# (c) Copyright IBM Corp. 2010, 2019. All Rights Reserved.
+# -*- coding: utf-8 -*-
+# pragma pylint: disable=unused-argument, no-self-use
+"""Function implementation"""
+
+import logging
+from resilient_circuits import ResilientComponent, function, handler, StatusMessage, FunctionResult, FunctionError
+import fn_phish_tank.util.selftest as selftest
+from resilient_lib import ResultPayload
+from fn_phish_tank.util.phish_tank_helper import *
+
+
+class FunctionComponent(ResilientComponent):
+    """Component that implements Resilient function 'fn_phish_tank_submit_url"""
+
+    def __init__(self, opts):
+        """constructor provides access to the configuration options"""
+        super(FunctionComponent, self).__init__(opts)
+        self.options = opts.get("fn_phish_tank", {})
+        selftest.selftest_function(opts)
+
+    @handler("reload")
+    def _reload(self, event, opts):
+        """Configuration options have changed, save new values"""
+        self.options = opts.get("fn_phish_tank", {})
+
+    @function("fn_phish_tank_submit_url")
+    def _fn_phish_tank_submit_url_function(self, event, *args, **kwargs):
+        """Function: Function to check URLs against PhishTanks(https://www.phishtank.com/) Database to see URL is flagged as phishing or not phishing."""
+        try:
+            # Get the function parameters:
+            phish_tank_check_url = kwargs.get("phish_tank_check_url")  # text
+
+            # Get function parameters from config file.
+            phish_tank_api_url = self.options.get('phish_tank_api_url')
+            phish_tank_api_key = self.options.get('phish_tank_api_key')
+            proxy = self.options.get('proxy')
+
+            log = logging.getLogger(__name__)
+            log.info("PhishTank Check URL: %s", phish_tank_check_url)
+            log.info("PhishTank API Access URL: %s", phish_tank_api_url)
+            log.info("PhishTank API Access KEY: %s", phish_tank_api_key)
+            log.info("Proxy Server Address: %s", proxy)
+
+            yield StatusMessage("Checking Phishing Status Against URL: {}".format(phish_tank_check_url))
+
+            # Initialing the resilient result object
+            _result_obj = ResultPayload('fn_phish_tank',
+                                        function_inputs=[phish_tank_api_url, phish_tank_check_url, phish_tank_api_key])
```

Comment:
```suggestion
                                        function_inputs=**kwargs)
```

Reply:
Instead of casting the 3 inputs to a list, I think you can just past in `**kwargs` which will be the function inputs. I can see you are passing in app.config values also. Are these function parameters? I don't think you should put things like API Key in the function payload. The idea of app.config is to prevent sending credentials as part of the function input and so we probably shouldn't send any as the function output.
Thoughts on this @sjcurtinibm ?

Reply:
File "/home/neetin/.virtualenvs/gRPC_3_6_4/lib/python3.7/site-packages/fn_phish_tank/components/fn_phish_tank_submit_url.py", line 48
    function_inputs=**kwargs)
                     ^
SyntaxError: invalid syntax

I declared in the same way got above error.

Reply:
My apologies I did a typo 

I think the above will work if we remove the function_inputs keyword like this 

```suggestion
                                        **kwargs)
```

Alternatively, function_inputs=kwargs may work. My main concern is that API key should not be a part of the result payload.

Reply:
ok